File: examples-sdk/go/024-basic-llm-chat/main.go
// 024-basic-llm-chat — Go SDK equivalent
// Compile: vil compile --from go --input 024-basic-llm-chat/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("llm-chat", 8080)

	chat := vil.NewService("chat")
	chat.Endpoint("POST", "/chat", "chat_handler")
	s.Service(chat)

	s.Compile()
}

File: examples-sdk/zig/024-basic-llm-chat/main.zig
// 024-basic-llm-chat — Zig SDK equivalent
// Compile: vil compile --from zig --input 024-basic-llm-chat/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("llm-chat", 8080);
    var chat = vil.Service.init("chat");
    chat.endpoint("POST", "/chat", "chat_handler");
    server.service(&chat);
    server.compile();
}
